rosco/controller/src/Controllers.f90

```diff
@@ -203,64 +204,84 @@ SUBROUTINE VariableSpeedControl(avrSWAP, CntrPar, LocalVar, objInst, ErrVar)
         CHARACTER(*),               PARAMETER           :: RoutineName = 'VariableSpeedControl'
 
         ! Allocate Variables
-        
+
         ! -------- Variable-Speed Torque Controller --------
         ! Define max torque
-        IF (LocalVar%VS_State == 4) THEN
+        IF (LocalVar%VS_State == VS_State_Region_3_ConstTrq) THEN
            LocalVar%VS_MaxTq = CntrPar%VS_RtTq
         ELSE
             ! VS_MaxTq = CntrPar%VS_MaxTq           ! NJA: May want to boost max torque
             LocalVar%VS_MaxTq = CntrPar%VS_RtTq
         ENDIF
-        
-        ! Optimal Tip-Speed-Ratio tracking controller
-        IF ((CntrPar%VS_ControlMode == 2) .OR. (CntrPar%VS_ControlMode == 3)) THEN
+
+        ! Optimal Tip-Speed-Ratio tracking controller (reference generated in subroutine ComputeVariablesSetpoints)
+        IF ((CntrPar%VS_ControlMode == VS_Mode_WSE_TSR) .OR. \
+            (CntrPar%VS_ControlMode == VS_Mode_Power_TSR) .OR. \
+            (CntrPar%VS_ControlMode == VS_Mode_Torque_TSR)) THEN
+
             ! Constant Power, update VS_MaxTq
-            IF (CntrPar%VS_ConstPower == 1) THEN
+            IF (CntrPar%VS_ConstPower == VS_Mode_ConstPwr) THEN
                 LocalVar%VS_MaxTq = min((CntrPar%VS_RtPwr/(CntrPar%VS_GenEff/100.0))/LocalVar%GenSpeedF, CntrPar%VS_MaxTq)
             END IF
 
+            IF (CntrPar%VS_FBP > 0) THEN
```

Comment:
This has been modified to instead be on line 213, but it does look like this is already the default. I added this to effectively reset the default because it's being changed in a lot of other conditional blocks in this function, so if anything should modify this variable unexpectedly, this would restore the default. But such a situation would probably be a larger bug that needs fixing anyway